README.md

```diff
@@ -23,3 +23,50 @@ These differences include:
    As much as possible the code includes comments about these extensions.
 
  - Many data fields not used by CoreUpdate are omitted.
+ 
+## `serve-package`
+
+This project includes a very simple program designed to serve a single Container Linux package on the local host. It is intended to be used as a manual updater for a machine that is not able to use a full-fledged CoreUpdate instance. Binaries are available for each released version on the [releases page](https://github.com/coreos/go-omaha/releases). `serve-package` can also be built from source using the provided Makefile:
+
+```bash
+make
+```
+
+The binary will be available in the `bin/` folder.
+
+It is recommended that the server be run directly on the machine you intend to update. Go to the [Container Linux releases notes](https://coreos.com/releases/) and find the version number for the release you would like to update to. The update payload can be retrieved from
+
+```
+https://update.release.core-os.net/amd64-usr/<version>/update.gz
+```
+
+where `<version>` is the version number you retrieved from the releases page. For example, `https://update.release.core-os.net/amd64-usr/1576.4.0/update.gz` is the payload required to update to Container Linux version 1576.4.0.
+
+Copy the update payload and the `serve-package` binary to the server you would like to update. `serve-package` can be run as follows:
+
+```bash
+./serve-package --package-file update.gz --package-version <version>
+```
+
+By default, the server listens on `localhost:8000`. This can be modified using the `--listen-address` option.
+
+Next, `update_engine` needs to be configured to use the local server that was just setup. Add the following line to `/etc/coreos/update.conf`:
```

Comment:
`set up` is two words in this case.

For clarity, maybe drop the second sentence?  We're not adding `echo "SERVER=...` to `update.conf`.

```
Next, configure `update_engine` to use the local server that was just set up:
```